.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: php
+
+php:
+  - '7.1'
+  - '7.2'
+  - '7.3'
+
+before_script:
+  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+  - chmod +x ./cc-test-reporter
+  - ./cc-test-reporter before-build
+  - travis_retry composer self-update
+  - travis_retry composer install --no-interaction --prefer-source
+
+script:
+  # Fail the Travis CI build on the first error.
+  - set -e
+  - mkdir -p build/logs
+  - php vendor/bin/phpcs --standard=PSR12 src
+  - php vendor/bin/phpcs --standard=PSR1 tests
+  - php vendor/bin/phpstan analyze --level max src
```

Comment:
I'm not interested in phpcs or phpstan at the moment, these lines can be removed